phpunit.xml.dist

```diff
@@ -6,15 +6,29 @@
 	convertNoticesToExceptions="true"
 	convertWarningsToExceptions="true"
 	>
+	<coverage processUncoveredFiles="false">
+		<include>
+			<directory suffix=".php">includes/</directory>
+			<file>plugin-check.php</file>
+		</include>
+	</coverage>
```

Comment:
Why do we need both this and the similarly looking `<filter>` bit below?

Reply:
PHPUnit requires such configuration to be able to collect code coverage. It complains otherwise.